assets/css/__style-guide.css

```
@@ -1,20 +1,20 @@
 :root {
-  --primary: #13a3fd;
-  --primary-light: #ffffff;
-  --primary-dark: #141414;
-  --text-white: #ffffff;
-  --text-grey: #8d8d8d;
+  --colors-primary: #13a3fd;
```

Use the singular prefix `--color-` to stay consistent with other prefixes (`--theme-`, `--font-`, `--text-`).

```suggestion
  --color-primary: #13a3fd;
```